Mathlib/Order/OmegaCompletePartialOrder.lean

```diff
@@ -553,14 +672,15 @@ instance : OrderHomClass (α →𝒄 β) α β where
 instance : PartialOrder (α →𝒄 β) :=
   (PartialOrder.lift fun f => f.toOrderHom.toFun) <| by rintro ⟨⟨⟩⟩ ⟨⟨⟩⟩ h; congr
 
-end
-
 namespace ContinuousHom
 
+protected lemma ωScottContinuous (f : α →𝒄 β) : ωScottContinuous f :=
+  ωScottContinuous.of_monotone_map_ωSup ⟨f.monotone, f.map_ωSup'⟩
+
 -- Not a `simp` lemma because in many cases projection is simpler than a generic coercion
 theorem toOrderHom_eq_coe (f : α →𝒄 β) : f.1 = f := rfl
 
-@[simp] theorem coe_mk (f : α →o β) (hf : Continuous f) : ⇑(mk f hf) = f := rfl
+@[simp] theorem coe_mk (f : α →o β) (hf) : ⇑(mk f hf) = f := rfl
```

Comment:
Let's keep the type-sig on `hf` for readability

Reply:
Done. It now becomes `(hf : ∀ (c : Chain α), f.toFun (ωSup c) = ωSup (c.map f))`.